### DISCUSSION TITLE
Generating wheels with one pyarmor installation

### DISCUSSION THREAD

**Discussion by jorgen** [dashingsoft/pyarmor#1465] [General]
Hi,
I want to generate wheels for all the targets (windows/linux/mac) and all the different python distributions we are targeting (3.7 -> 3.11). Can I generate different runtimes for the different targets, store them and use them when obfuscating my code? When I pack up the different wheel files I could just include the correct runtime?

I have looked in the documentation and I cant find anything pointing me in the right direction. Also should we have the group license for this, or is the pro license ok?

**Comment by jondy** [comment]
First generate runtime files by `pyarmor gen runtime`, use option `-h` to check help.
Then generate the obfuscated scripts by `pyarmor gen --use-runtime /path/to/pre-built-runtime ...`
Finally build wheels as normal scripts.

Either group license or pro license is OK.

**Reply by jorgen** [reply]
Thank you for your response. If I do as you suggest above I get this output:

![image](https://github.com/dashingsoft/pyarmor/assets/30806/84cc9240-60e0-4354-9972-bda0773eb7f5)

It doesn't matter if I use -i or not.
